vignettes/stat_specs.Rmd

```diff
@@ -27,72 +28,79 @@ knitr::opts_chunk$set(
 
-In practice, treatment discontinuation rates may be relatively low or it may be difficult to retain patients in the trial after the ICE. In such settings, the amount of available post treatment discontinuation data may be insufficient to inform the imputation model. Depending on the disease area and the anticipated mechanism of action of the intervention, it may be plausible to assume that subjects in the intervention group behave similarly to subjects in the control group after the ICE treatment discontinuation. In this case, reference-based imputation methods are an option [@Mallinckrodt2020]. Reference-based imputation methods are increasingly popular for handling missing data under a missing-not-at-random (MNAR) assumption. They formalize the idea to impute missing data in the intervention group based on data from a control or reference group. For a general description and review of reference-based imputation methods, we refer to @CarpenterEtAl2013, @CroEtAlTutorial2020, and @Wolbers2021. For a technical description of the implemented statistical methodology for reference-based imputation, we refer to section \@ref(sec:statsMethods) (in particular section \@ref(sec:imputationStep)). 
+In practice, treatment discontinuation rates may be relatively low or it may be difficult to retain patients in the trial after the ICE. In such settings, the amount of available post treatment discontinuation data may be insufficient to inform the imputation model. Depending on the disease area and the anticipated mechanism of action of the intervention, it may be plausible to assume that subjects in the intervention group behave similarly to subjects in the control group after the ICE treatment discontinuation. In this case, reference-based imputation methods are an option (@Mallinckrodt2020). Reference-based imputation methods formalize the idea to impute missing data in the intervention group based on data from a control or reference group. For a general description and review of reference-based imputation methods, we refer to @CarpenterEtAl2013, @CroEtAlTutorial2020, @White2020causal and @Wolbers2021. For a technical description of the implemented statistical methodology for reference-based imputation, we refer to section \@ref(sec:statsMethods) (in particular section \@ref(sec:imputationStep)). 
```

Comment:
> In practice, treatment discontinuation rates may be relatively low or it may be difficult to retain patients 

Apologies I found this phrasing confusing confusing, would it be worth re-wording it? @wolbersm

Reply:
Hi @gowerc 
How about the following instead?

> In some trial settings, only few subjects discontinue the randomized treatment. In other settings, treatment discontinuation rates are higher but it is difficult to retain subjects in the trial discontinuation leading to sparse data collection after treatment discontinuation. In both settings, the amount of available data after treatment discontinuation may be insufficient to inform an imputation model which explicitly models post-discontinuation data. Depending on... [...]

Reply:
That is significantly clearer to me :)